Package.swift

```diff
@@ -27,33 +27,49 @@ let package = Package(
         .library(name: "SpeziValidation", targets: ["SpeziValidation"])
     ],
     dependencies: [
+        .package(url: "https://github.com/StanfordSpezi/Spezi.git", from: "1.3.0"),
         .package(url: "https://github.com/apple/swift-collections.git", from: "1.1.0"),
-        .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.15.3")
+        .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.15.3"),
+        .package(url: "https://github.com/realm/SwiftLint.git", .upToNextMinor(from: "0.55.1"))
```

Comment:
Similar question as in the Spezi PR review: https://github.com/StanfordSpezi/Spezi/pull/106. I would rather suggest to add SwiftLint as a dependency to the main UI Testing Application and Xcode project to avoid a lengthy SPM build process and transitive dependency resolution if possible. Could be resolved and should be integrated in the future one SPM is smarter about dependency resolution & what dependencies should be built.

Reply:
Same proposal as in Spezi, we use the `SPEZI_DEVELOPMENT_SWIFTLINT` environment variable to conditionally depend and enable the SwiftLint build tool.

Reply:
Sounds great!